test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -822,6 +822,11 @@ public class IRNode {
         beforeMatchingNameRegex(MAX, "Max(I|L)");
```

Comment:
For completeness, we should probably also add `F` and `D` here. I've quickly checked the usages uf `MAX` and we only seem to be using it in `failOn` conditions. So, this should be safe to do.